xml/System.Security.Cryptography.Pkcs/Pkcs12Builder.xml

```diff
@@ -241,11 +314,25 @@
         <Parameter Name="iterationCount" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="password">To be added.</param>
-        <param name="hashAlgorithm">To be added.</param>
-        <param name="iterationCount">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="password">The password to use as a key for computing the MAC.</param>
+        <param name="hashAlgorithm">The hash algorithm to use when computing the MAC.</param>
+        <param name="iterationCount">The iteration count for the Key Derivation Function (KDF) used in computing the MAC.</param>
+        <summary>Seals the PFX against further changes by applying a password-based Message Authentication Code (MAC) over the contents with a password from a span.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+ Any hash algorithm that this method accepts can be processed by the <xref:System.Security.Cryptography.Pkcs.Pkcs12Info> class.
+ Some choices of hash algorithm may cause failures in <xref:System.Security.Cryptography.X509Certificates.X509Certificate2..ctor%2A> or <xref:System.Security.Cryptography.X509Certificates.X509Certificate2Collection.Import%2A> on some Operating Systems, as these methods depend on system libraries for support.
```

Comment:
```suggestion
 Some choices of hash algorithm may cause failures in <xref:System.Security.Cryptography.X509Certificates.X509Certificate2.%23ctor%2A> or <xref:System.Security.Cryptography.X509Certificates.X509Certificate2Collection.Import%2A?displayProperty=nameWithType> on some operating systems, because these methods depend on system libraries for support.
```